src/lib/client/store/activities.ts

```diff
@@ -33,8 +33,24 @@ export function buildActivityTypeSelector(missions: boolean) {
   return (state: RootState) => state.activities.list.filter(f => f.isMission === missions);
 }
 
-export function getActiveParticipants(a: Activity) {
-  return Object.values(a.participants).filter(p => (p.timeline.slice(-1)?.[0].status ?? ResponderStatus.Unavailable) & 0x01);
+export function getActiveParticipants(activity: Activity) {
+  return filterParticipantsByStatus(Object.values(activity.participants), [ResponderStatus.SignedIn])
+}
+
+export function getSignedInCount(activity: Activity) {
+  return filterParticipantsByStatus(Object.values(activity.participants), [ResponderStatus.SignedIn]).length;
+}
+
+export function getStandbyCount(activity: Activity) {
+  return filterParticipantsByStatus(Object.values(activity.participants), [ResponderStatus.Standby]).length;
+}
+
+export function getSignedOutCount(activity: Activity) {
+  return filterParticipantsByStatus(Object.values(activity.participants), [ResponderStatus.SignedOut]).length;
+}

```

Comment:
I'm kind of thinking "you ain't gonna need it" here. I'd guess there's probably even odds that these implementations are going to be what we need next (vs mixing statuses, etc). Until then, we have to maintain methods with nothing pointing to them.

Reply:
Fair. I have some ideas for more EventTile data, but I will delete them for now. Easy enough to re-implement when we need them.